CHANGELOG.md

```diff
@@ -1,5 +1,20 @@
 # Changelog
 
+## 1.10.4 - 2024-07-08
+
+### Added
+
+- `Swift` terminal profile
+- Set Swift environment variables in the integrated terminal
+- Show macro expansions
```

Comment:
Also give a shoutout to @lokesh-tr for his great work on this!

Reply:
Removed